src/main/java/com/linkedin/replica/jobs/database/handlers/impl/ArangoSQLJobsHandler.java

```diff
@@ -87,7 +97,7 @@ public void deleteAll(){
 
     public List<Job> getSavedJobs(String userID){
         try {
-            User user= dbInstance.collection("jobs").getDocument(userID,User.class);
+            User user = dbInstance.collection("jobs").getDocument(userID,User.class);
```

Comment:
1. Use the config for collection name
2. Why are you getting the user you have?

3. what do you return on success?